inventory/local_site_specific_vars.yml.example

```diff
@@ -0,0 +1,31 @@
+# The license key for the Geo API:
+# https://cvmfs.readthedocs.io/en/stable/cpt-replica.html#geo-api-setup
+# For some unclear reason, the Stratum 1 installation fails when this is not set:
+# https://github.com/EESSI/filesystem-layer/issues/2
+cvmfs_geo_license_key: INSERT_YOUR_KEY
```

Comment:
nitpicking here but to follow the same convention every site specific variable starts by `local_`

Reply:
same here. it's a example file so it's always site-specific. please ignore the comment. looks perfect

Reply:
Okay! I'm not completely sure about the local_ anyway: if we want to support that for all variables, we would need extra code in the group_vars/all.yml to pick up these variables.

Reply:
I think it makes sense to follow same approach for every var defined in `inventory/local_site_specific_vars.yml` but it's just a small detail. Do it as you prefer